packages/react-native/ReactCommon/react/renderer/components/view/conversions.h

```diff
@@ -556,6 +556,36 @@ inline void fromRawValue(
   result = transformMatrix;
 }
 
+ inline void fromRawValue(
+     const PropsParserContext &context,
+     const RawValue &value,
+     TransformOrigin &result) {
+   react_native_expect(value.hasType<std::vector<RawValue>>());
+   auto origins = (std::vector<RawValue>)value;
+   
+   TransformOrigin transformOrigin;
+
+   size_t i = 0;
+   for (const auto& origin : origins) {
```

Comment:
I think we need to be careful because if the user somehow managed to pass in more than 3 values, we'd get a crash setting `transformOrigin.origin[3]`

Reply:
added an expect check, that origins should always be equal to 3! https://github.com/facebook/react-native/pull/38559/commits/302a9aaf283a2d05de01ea314e996fb848c89bcb